Cython/Build/Dependencies.py

```diff
@@ -881,7 +881,7 @@ class Extension_setuptools(Extension): pass
 
 # This is the user-exposed entry point.
 def cythonize(module_list, exclude=None, nthreads=0, aliases=None, quiet=False, force=False, language=None,
-              exclude_failures=False, **options):
+              exclude_failures=False, replace_extension=False, **options):
```

Comment:
`replace_extension` isn't the ideal name because it reads in this context like we're replacing the extension (module), not the (file) extension. Maybe `auto_use_pyx`? And that should obviously be documented in the compilation docs.

Reply:
I've changed it to `auto_use_pyx` and documented it in the docstring. Anywhere else that I missed?